Toolset/palettes/revdatagridlibrary/behaviorsdatagridbuttonbehavior.livecodescript

```diff
@@ -368,7 +368,7 @@ private command _CreateMobileScroller
       mobileControlSet sScrollerId, "canBounce", "true"
       mobileControlSet sScrollerId, "pagingEnabled", "false"
       mobileControlSet sScrollerId, "canScrollToTop", "true"
-      mobileControlSet sScrollerId, "delayTouches", "false"
+      mobileControlSet sScrollerId, "delayTouches", "true"
```

Comment:
Should this rather be merged into develop-9.6 to up its priority/urgency?

Reply:
Well, I didn't see a 9.6 branch open and
nobody else commented on the how list and
nobody else commented on the bug report and
this has been like this since whenever dg2 came out but nobody else seemed to notice, so I'm going with "maybe".
I mean, I personally think it should be a hotfix and get a special code name and a bug bounty Bond car to me and a lifetime free subscription to every LC premium thing, but I'm also megalomaniac.

Reply:
I agree with Mikey: this bug is driving me nuts.

Reply:
Don't hold your breath waiting for a pull request to be reviewed.
Meanwhile, this is a stack-only script, so you can always patch your own copy.